sdk/core/azure-core/inc/azure/core/operation.hpp

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include "azure/core/context.hpp"
+#include "azure/core/operation_status.hpp"
+#include "azure/core/response.hpp"
+
+#include <chrono>
+#include <string>
+#include <memory>
```

Comment:
```suggestion
#include <memory>
#include <string>
```

Reply:
Or whatever order Ctrl+K Ctrl+D in VS formats this as :)